keyboards/handwired/tritium_numpad/readme.md

```diff
@@ -0,0 +1,26 @@
+Tritium Numpad
+===
+
+Keyboard Maintainer: QMK Community  
+Hardware Supported: Handwired 6x4 numpads using promicro controller
```

Comment:
```suggestion
Hardware Supported: Handwired 6x4 numpads using promicro controller  
```

Reply:
was the space here added on accident?

Reply:
It's necessary for proper rendering in Markdown. Double spaces at the end = newline.